lib/debug.nix

```diff
@@ -95,17 +94,20 @@ rec {
        => null
    */
   traceSeqN = depth: x: y: with lib;
-    let snip = v: if      isList  v then noQuotes "[…]" v
-                  else if isAttrs v then noQuotes "{…}" v
-                  else v;
-        noQuotes = str: v: { __pretty = const str; val = v; };
-        modify = n: fn: v: if (n == 0) then fn v
-                      else if isList  v then map (modify (n - 1) fn) v
-                      else if isAttrs v then mapAttrs
-                        (const (modify (n - 1) fn)) v
-                      else v;
+    let
+      snip = v:
+        if isList v then noQuotes "[…]" v
+        else if isAttrs v then noQuotes "{…}" v
+        else v;
+      noQuotes = str: v: { __pretty = const str; val = v; };
+      modify = n: fn: v:
+        if (n == 0) then fn v
+        else if isList v then map (modify (n - 1) fn) v
+        else if isAttrs v then mapAttrs
+          (const (modify (n - 1) fn)) v
+        else v;
     in trace (generators.toPretty { allowPrettyValues = true; }
-               (modify depth snip x)) y;
+      (modify depth snip x)) y;
```

Comment:
How to handle this? the indentation doesn't match. Should there be a newline after `in`?